pkg/apis/apiserver/corev1alpha.conversion.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 The Bulward Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiserver
+
+import (
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"sigs.k8s.io/apiserver-builder-alpha/pkg/builders"
+
+	corev1alpha1 "github.com/kubermatic/bulward/pkg/apis/core/v1alpha1"
+)
+
+func init() {
+	utilruntime.Must(Corev1alpha1RegisterConversion(builders.Scheme))
+	utilruntime.Must(RegisterConversions(builders.Scheme))
+	utilruntime.Must(RegisterDefaults(builders.Scheme))
```

Comment:
Why do we use `builders.Scheme` not `localSchemeBuilder`?

Reply:
done